kde-misc/kolor-manager/kolor-manager-9999.ebuild

```diff
@@ -15,7 +15,8 @@ SLOT="4"
 IUSE="debug"
 
 DEPEND="
-	>=media-libs/oyranos-0.9.5
+	media-gfx/synnefo[qt4]
+	=media-libs/oyranos-9999
```

Comment:
According to upstream yes, since recently. Last oyranos release was more than one year ago.